### PR TITLE
Add Django project template with some configuration changes

### PR DESCRIPTION
**Pull request by justwheel**
This pull request combines the different building blocks of our project. Some configuration changes are also made, explained by each commit:

1. 866a4847bac85ba25a0438d788fd0e86e68d5a5a: Imports starter template almost directly from upstream. Few to no changes made.
2. fc93f035afdd4665958423f67c6fb1285f725b59: Lock our project to Python 3.6. This is done to standardize on a single version of Python. It is also the version of Python 3 most easily installed on CentOS 7 hosts (used by Monroe County).
3. 46b207fd71eba0f1ce4b903027d5ce38f2218f8b: Remove a config file we do not need.
4. 4a39c3672c6e908c7a479dbe5bf9bbede216f3e3: Set time zone to New York since we will only serve one geographic area.
5. e587c2cce0a3a0e4c950d6718f0367130eb4bdf0: Consistently link to the correct version of Django docs in all comments used in the project, to make sure we all use the same documentation to build our app.

### CODE REVIEW

File: config/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# Set another default user than root for security reasons
```
Comment (justwheel) [inline config/nginx.conf:1]: @nlMeminger The project template provides a nginx config file. Might be useful for you to look through.

File: config/supervisord.conf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+[program:uwsgi]
+command = env/bin/uwsgi -ini config/uwsgi.ini
+autostart = true
+autorestart = true
+redirect_stderr = true
+stopsignal = QUIT
+
+[supervisord]
```
Comment (justwheel) [inline config/supervisord.conf:8]: @nlMeminger It also ships with a supervisord config we could use for application health checks. We don't have time to create monitoring infrastructure, but we can make it easy to integrate into monitoring architectures where possible.

File: apps/base/urls.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+"""urlconf for the base application"""
+
+from django.conf.urls import url
+
+from .views import home
+
+
+urlpatterns = [
+    url(r'^$', home, name='home'),
+]
```
Comment (justwheel) [inline apps/base/urls.py:10]: Noting from back-end meeting, @nlMeminger mentioned this is a deprecated way of handling url schemas.

File: grasa_event_locator/settings/production.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.',
+        'NAME': '',
+        'USER': '',
+        'PASSWORD': '',
+        'HOST': '',
+        'PORT': '',
+    },
+}
```
Comment (justwheel) [inline grasa_event_locator/settings/production.py:17]: @eguy006 @leong96 This is where we will eventually configure a "production" MySQL server. Later on, we will have actual database credentials for our staging and production infrastructure.

File: grasa_event_locator/settings/production.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+# WSGI SETTINGS
+# https://docs.djangoproject.com/en/1.10/ref/settings/#wsgi-application
+WSGI_APPLICATION = '{{ project_name }}.wsgi.application'
+
+# NOTIFICATIONS
+# A tuple that lists people who get code error notifications.
+# https://docs.djangoproject.com/en/1.10/ref/settings/#admins
+ADMINS = (
+         ('Your Name', 'your_email@example.com'),
```
Comment (justwheel) [inline grasa_event_locator/settings/production.py:34]: Note: Need to fetch a persistent contact address for admin error notifications. Does a monitoring system exist? Or better, can we configure this with environment variables.

File: apps/base/views.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+"""Views for the base app"""
```
Comment (justwheel) [inline apps/base/views.py:1]: @leong96 @eguy006 @nlMeminger The "base" app will be the search functionality. In this case, we want to make a copy of the `base/` app, probably as `events/`. Then we will work on building out the event curation system.

**Comment by justwheel** [comment]
I left some commits on my desktop. :disappointed: I'm going to push them up tonight or tomorrow morning and put this up for review.

**Comment by leong96** [comment]
All comments understood.

**Review by leong96 — APPROVED** [review]

**Pull request merged**